### DISCUSSION TITLE
We are planning an organization name change on September 20

### DISCUSSION THREAD

**Discussion by IgorTodorovskiIBM** [zopencommunity/meta#860] [Announcements]
Hi folks, we would like to share an important update regarding our z/OS Open Tools GitHub organization. As we work towards onboarding to the Open Mainframe Project, we’ll be **renaming our initiative from z/OS Open Tools to the zopen community (https://github.com/zopencommunity)**. We plan to implement this change on **September 20** to better align with IBM's IP guidelines regarding the use of the term "z/OS."

**Here’s what you will need to know:**

**Automatic Changes:**
* GitHub will redirect references to the zosopentools repositories, so existing links will continue to work.
* You can push to the old remote URL, but we recommend updating it after the name change to avoid potential future conflicts.

**Non-Automatic Changes:**
* Links to the old organization profile will return a 404 error. We’ll update all existing links within the z/OS Open Tools org on Sept. 20, but please update any org links in blogs, social media, etc.
* API requests using the old organization name will also result in 404 errors, so update those accordingly - we will transition all of our scripts to use the new org name. Note, the zopen package manager does not make any github API requests.

For more details on what's affected, see [GitHub’s guide on renaming an organization](https://docs.github.com/en/organizations/managing-organization-settings/renaming-an-organization).

While this change may feel disruptive, joining the Open Mainframe Project will bring several benefits:
* Improved Collaboration: The Open Mainframe Project provides additional resources and support that will help accelerate our initiative’s progress.
* Increased Visibility: As part of the Linux Foundation through OMP, our project’s profile will be raised, attracting new contributors and users.
* Additional Resources: OMP will provdie additional assets such as a new logo, domain, and paid services like security scanning and Github Team access. 

If you have any comments/concerns, please let us know. Again, we plan to make the transition to https://github.com/zopencommunity on September 20. Thanks!

**Comment by IgorTodorovskiIBM** [comment]
The community has been renamed to https://github.com/zopencommunity (and not zopen-community as mentioned earlier due to technical issues). We're still going through our checklist, so please anticipate some issues. We'll make an announcement when the transition is completed.

**Comment by v1gnesh** [comment]
Looks like `ZOSOpenTools` needs to be replaced with `zopencommunity` in all ports' `.github/actions/{build_and_test,bump}.yml` files.
`multi-gitter` from the CI machine should be able to do this.

**EDIT:** With write access to all repos, it [can do](https://github.com/lindell/multi-gitter?tab=readme-ov-file#usage) both - `run` to create PRs, and `merge` to merge them.
**EDIT2:** For selfish reasons, I'm sending PRs via mig right now :)
